docs/Using Price Feeds/get-the-latest-price-solana.md

```diff
@@ -170,7 +170,7 @@ The example code operates using the following process:
 
 1. Your deployed program receives the request and starts processing at the [`entrypoint` in `src/lib.rs`](https://github.com/smartcontractkit/chainlink-solana-demo/blob/main/src/lib.rs#L45). This is the Rust program that you built and deployed to the Solana Devnet.
 
-1. The `process_instruction` function in `lib.rs` receives the transaction with the specified feed address and handles the steps to retrieve and store the price data on-chain. The function also calls `get_price()` from the [chainlink-solana](https://github.com/smartcontractkit/chainlink-solana) package, which gets imported from GitHub in the `Cargo.toml` file. See the [`Cargo.toml`](https://github.com/smartcontractkit/chainlink-solana-demo/blob/main/Cargo.toml#L19) file, which maps the `chainlink-solana` package to `chainlink`.
+1. The `process_instruction` function in `lib.rs` receives the transaction with the specified feed address and handles the steps to retrieve and store the data on-chain. The function also calls `get_price()` from the [chainlink-solana](https://github.com/smartcontractkit/chainlink-solana) package, which gets imported from GitHub in the `Cargo.toml` file. See the [`Cargo.toml`](https://github.com/smartcontractkit/chainlink-solana-demo/blob/main/Cargo.toml#L19) file, which maps the `chainlink-solana` package to `chainlink`.
```

Comment:
This one is explicitly price data (see `get_price`), so we could consider leaving it, but I think it's okay to change anyway.

Reply:
I was 50/50 on making this change as well. I'll revert it.